_notebooks/2021-02-09-task-spooler.ipynb

```diff
@@ -0,0 +1,2746 @@
+{
```

Comment:
<p>add <code>%%capture</code>  to remove output</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LibreCV/blog/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>